ATAPHtmlReport/ATAPHtmlReport.psm1

```diff
@@ -579,6 +579,96 @@ function Merge-CisAuditsToMitreMap {
 	}
 }
 
+function Get-MitigationsFromFailedTests {
+	param(
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        $Mappings
+    )
+	Begin {
+		$json = $global:CISToAttackMappingData.'CISAttackMapping'
+		$CISAMitigationsFromPaper = [ordered]@{
+			'M1017' = @{
+				'MitreTechniqueIDs' = @()
```

Comment:
muessen wir die MitreTechniqueIDs fuer jede taktik bei jedem ausfuehren des reports suchen? sind das nicht immer die gleichen, dann koennte das doch in der json statisch stehen?